packages/pantheon/src/index.ts

```diff
@@ -0,0 +1,177 @@
+import sower from 'sower';
+import seedrandom from 'seedrandom';
+
+import pick from './utils/pick';
+import d from './utils/d';
+import make from './utils/make';
+import mediator from './utils/mediator';
+
+import { createDeity, Deity } from './deity';
+import { generateDisposition, Disposition } from './dispositions';
+import { reproduce } from './reproduce';
+
+// TODO: workshop these. I like them conceptually, especially for competing
+// pantheons, but need to think through implications/balance
+const ages = ['ancient', 'new', 'old', 'original', 'young'] as const;
+
+interface Pantheon {
+  dispositions: Array<Disposition>;
+  age: typeof ages[number];
+  chief?: string; // deity.id
+  chiefSpouse?: string;
+  deities: Array<Deity>;
+  seed: string;
+}
+
+const createPantheon = (seed: string = sower.silly()): Pantheon => {
+  // seed our random number generator
+  mediator.provide('random', seedrandom(seed));
+
+  const deities = [];
+  const dispositions = make(3).map(() => generateDisposition());
+  const relationships = [];
+
+  // Step 1: Create a chief deity
+  const chief = createDeity(dispositions, []);
+  relationships.push({
+    first: 'anon-chronos',
+    second: chief.id,
+    action: 'beget',
+  });
+
+  // Step 2: Create a "sideboard" of deities. These are
+  // potential suitors for the second generation and potential
+  // relatives of the chief, children of titans and eldritch horrors.
+  const sideBoard = make(10).reduce((acc, _, idx) => {
```

Comment:
any reason we want exactly ten sideboard deities instead of randomizing it as well?

Reply:
Ohhhh hold up, this is actually a change you made to make. 
Make’s second argument, originally seed, determined if it was making an array of random size (with max number) or fixed size

Reply:
oh wait, i've been misreading this entirely then lol i was construing make with array.make or whatever that auto generated length array function is. my bad. disregard